gradle/libs.versions.toml

```diff
@@ -114,7 +119,7 @@ tink = { module = "com.google.crypto.tink:tink-android", version.ref = "tink" }
 coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 
 kotlin-bom = "org.jetbrains.kotlin:kotlin-bom:1.8.0"
-compose-bom = "androidx.compose:compose-bom:2023.05.01"
+compose-bom = "androidx.compose:compose-bom:2024.06.00"
```

Comment:
This is the one notable change that RevenueCat should check. Emerge snapshots requires the `foundation-android` dependency, which would not resolve without updating the BOM version

Reply:
Right we will need to verify that there are no unintended changes. We're stuck with an older version of the compose compiler since we can't update kotlin without having issues in some of our hybrid SDKs. But I believe updating the library itself *should* work.

Reply:
Let me know how I can help test that. I can open this PR against the main purchases-android repository and we can let integration tests run, and in addition if there's any local running or validation I can perform I'm happy to do so!